### PR TITLE
Isochrones: forbidden_uris fix bug

### PR DESCRIPTION
**Pull request by ZoeBrunet**

### CODE REVIEW

File: source/jormungandr/tests/graphical_isochrones_tests.py [hunk 1]
```diff
@@ -219,6 +219,21 @@ def test_graphical_isochrone_multi_isochrone(self):
...
+
+        is_valid_graphical_isochrone(basic_response, self.tester, basic_query)
+        is_valid_graphical_isochrone(response_forbbiden_uris, self.tester, basic_query)
+
+        for basic_isochrone, isochrone_forbbiden_uris in zip(basic_response['isochrones'],
+                                                           response_forbbiden_uris['isochrones']):
+            basic_multi_poly = asShape(basic_isochrone['geojson'])
+            multi_poly_forbidden_uris = asShape(isochrone_forbbiden_uris['geojson'])
+
+        assert basic_multi_poly.contains(multi_poly_forbidden_uris)
```
Comment (TeXitoi) [inline source/jormungandr/tests/graphical_isochrones_tests.py]:
``` python
assert not multi_poly_forbidden_uris.contains(basic_multi_poly)
```

to be sure they are different.

File: source/jormungandr/tests/graphical_isochrones_tests.py [hunk 2]
```diff
@@ -219,6 +219,25 @@ def test_graphical_isochrone_multi_isochrone(self):
 
         assert len(response['isochrones']) == 7
 
+    def test_graphical_isochrones_forbidden_uris(self):
+        basic_query = "v1/coverage/main_routing_test/isochrones?from={}&datetime={}&max_duration={}"
+        basic_query = basic_query.format(s_coord, "20120614T080000", "300")
+        basic_response = self.query(basic_query)
+        q_forbidden_uris = basic_query + "&forbidden_uris[]=A"
+        q_forbidden_uris = q_forbidden_uris.format(s_coord, "20120614T080000", "300")
```
Comment (TeXitoi) [inline source/jormungandr/tests/graphical_isochrones_tests.py]: You don't need this line.

File: source/jormungandr/tests/graphical_isochrones_tests.py [hunk 3]
```diff
@@ -219,6 +219,25 @@ def test_graphical_isochrone_multi_isochrone(self):
...
+
+        is_valid_graphical_isochrone(basic_response, self.tester, basic_query)
+        is_valid_graphical_isochrone(response_forbidden_uris, self.tester, basic_query)
+
+        for basic_isochrone, isochrone_forbidden_uris in zip(basic_response['isochrones'],
+                                                             response_forbidden_uris['isochrones']):
+            basic_multi_poly = asShape(basic_isochrone['geojson'])
+            multi_poly_forbidden_uris = asShape(isochrone_forbidden_uris['geojson'])
+
+        assert not basic_multi_poly.contains(multi_poly_forbidden_uris)
```
Comment (TeXitoi) [inline source/jormungandr/tests/graphical_isochrones_tests.py]: Why not? with only a max_duration and less vehicle, your shape must be included, no?

Comment (ZoeBrunet) [inline source/jormungandr/tests/graphical_isochrones_tests.py]: The two polygons have a border in common thus contains() return false.

Comment (TeXitoi) [inline source/jormungandr/tests/graphical_isochrones_tests.py]: OK, then this assert is quite useless and missleading, just remove it.

**Pull request merged**